### DISCUSSION TITLE
output of .sketch.ask as a variable.

### DISCUSSION THREAD

**Discussion by dbigman** [approximatelabs/sketch#34] [Q&A]
I want to capture the output of .sketch.ask as a variable. Example:
---
> import sketch
> 
> OPENAI_API_KEY=OPENAI_API_KEY_SKETCH
> SKETCH_USE_REMOTE_LAMBDAPROMPT=False
> 
> user_answer = pivoted_data.sketch.ask(user_question)

This data has 8 columns.

---

however, the user_answer variable is empty and type 'NoneType'. How can I get: 

> user_answer = 'This data has 8 columns.'

**Comment by bluecoconut** [comment]
Add `call_display=False` to your function

so, 
```
user_answer = pivoted_data.sketch.ask(user_question, call_display=False)
```
